Assets/MixedRealityToolkit/Utilities/Editor/EditorProjectUtilities.cs

```diff
@@ -246,7 +256,6 @@ private static DirectoryInfo GetPackageCache()
             DirectoryInfo[] folders = root.GetDirectories(folderName);
             if (folders.Length == 0) 
             {
-                Debug.Log($"Failed to locate a folder fitting the {folderName} pattern.");
```

Comment:
Why remove this?

Reply:
It lead to confusion and concern when customers chose to _not_ include ar foundation in their project.

Reply:
i got a question about that line this morning, in fact.